exercises/G-conditionals-2/exercise-2.js

```diff
@@ -7,7 +7,13 @@
 
 */
 
-function studentPassed(grade) {}
+function studentPassed(grade) {
+	if (grade < 50) {
+		return 'failed'
+	} else if (grade >= 50) {
+		return 'passed'
+	}
+}
```

Comment:
If a number is not less than 50 it is always more than or equal to 50 instead. So you can remove the second condition here as it will always be true.

if (grade < 50) {
    return "failed"
} else {
    return "passed
}

Reply:
Thank you, this does make sense.